src/Symfony/Component/Validator/Tests/Constraints/CardSchemeValidatorTest.php

```diff
@@ -46,6 +46,36 @@ public function testValidNumbers($scheme, $number)
         $this->assertNoViolation();
     }
 
+    /**
+     * @requires PHP 8
+     *
+     * @dataProvider getValidNumbers
+     */
+    public function testValidNumbersWithNewLinePhp8($scheme, $number)
```

Comment:
since this test case is the one that's going to last, it shouldn't have the "php8" term embeded

Reply:
Updated